drivers/dma/Kconfig.stm32

```diff
@@ -15,20 +19,20 @@ if DMA_STM32
 
 config DMA_STM32_V1
 	bool
-	depends on SOC_SERIES_STM32F2X || SOC_SERIES_STM32F4X || SOC_SERIES_STM32F7X
+	default $(DT_COMPAT_ST_STM32_DMA_V1)
 	help
```

Comment:
I think the help strings should be changed too

Reply:
Indeed, done